### ISSUE TITLE
bower is deprecated

### ISSUE THREAD

**Issue by enpointe** [udacity/exoplanet-explorer#21]
This project is seriously out of date. These project is referenced in the Udacity Java Promises class. Being so out of date students have to struggle to attempt to fix and cobble things together in an attempt to get things working. If they are even able to.

Using out of date components makes it difficult to get things to work

npm WARN deprecated bower@1.8.8: We don't recommend using Bower for new projects. Please consider Yarn and Webpack or Parcel. You can read how to migrate legacy project here: https://bower.io/blog/2017/how-to-migrate-away-from-bower/

This code base should be updated to make it possible for students to follow the directions the JavaScript Promises course outlines.

**Comment by PraneetDixit** [comment]
@enpointe yes. I got several `npm WARN depreciated` statements while installing npm packages. Also, the `gulpfile.js` is not configured properly to be used with gulp 4.0.2. I simply cannot start the project because I get tonnes of errors when I run `gulp serve` command. The codebase needs urgent and serious update.

**Comment by PraneetDixit** [comment]
The repository was last updated in 2015 and it has been 5 years since then and a lot have things have changed. Several dependencies used in this project are depreciated. I even get warnings that some things may break out in `node 14.x` and many dependencies may have serious security issues. @cameronwp kindly look into this matter and do the needful as soon as possible. We are not able to study further in this course because the project would simply not start.
